.phpstorm.meta.php

```diff
@@ -15,6 +15,7 @@
 			'admin.paired_browsing'              => \AmpProject\AmpWP\Admin\PairedBrowsing::class,
 			'admin.plugin_row_meta'              => \AmpProject\AmpWP\Admin\PluginRowMeta::class,
 			'admin.polyfills'                    => \AmpProject\AmpWP\Admin\Polyfills::class,
+			'admin.rest_preloader'               => \AmpProject\AmpWP\Admin\RESTPreloader::class,
```

Comment:
This is missing the second service that was added:
```php
'validated_url_rest_controller' => \AmpProject\AmpWP\Validation\ValidatedUrlRESTController::class,
```

Reply:
Good catch. I've added it back in 26525f8.